### PR TITLE
Add I2K meeting

### PR DESCRIPTION
**Pull request by jburel**
cc @pwalczysko

### CODE REVIEW

File: events/index.html [hunk 1]
```diff
@@ -8,6 +8,16 @@
         <!-- begin Events -->
         <hr class="whitespace">
         <div id="community-posts" class="row">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-calendar"></i> November 03, 2018</h6>
```
Comment (mtbc) [inline events/index.html]: is December?

File: events/index.html [hunk 2]
```diff
@@ -8,6 +8,16 @@
...
+                <h2><a href="https://www.embl.de/training/events/2018/IMJ18-01/" target="_blank">OMERO Workshop, Heidelberg</a></h2>
+                <p class="card-caption">a 1h30 workshop on Data Management with OMERO.</p>
```
Comment (mtbc) [inline events/index.html]: initial capital A, to follow existing styling

**Comment by mtbc** [comment]
See https://snoopycrimecop.github.io/www.openmicroscopy.org/index.html for staging.

**Comment by jburel** [comment]
Merging so it is included as part of 5.4.9 deployment

**Pull request merged**